Lib/test/test_sax.py

```diff
@@ -397,6 +402,13 @@ def test_string(self):
         self.checkContent(prep.getByteStream(),
                           b"This was read from a file.")
 
+    def test_path_objects(self):
+        # If the source is a Path object, use it as a system ID and open it.
+        prep = prepare_input_source(pathlib.Path(self.file))
```

Comment:
Use `test.support.FakePath` instead of `pathlib.Path` for testing.

Reply:
Done :)